js/CXGN/List.js

```diff
@@ -753,9 +753,10 @@ CXGN.List.prototype = {
         }
     },
 		seedlotSearch: function(list_id){
+			var self = this;
 			jQuery('#working_modal').modal('show');
-			var accessions = this.getList(list_id);
-			console.log(accessions);
+			var data = this.getListData(list_id);
```

Comment:
should use the getList function instead of getListData if you are just getting the contents